neuro_config_client/client.py

```diff
@@ -276,6 +279,159 @@ async def remove_storage(
                 raise
         return await self.get_cluster(cluster_name)
 
+    async def get_node_pool(
+        self,
+        cluster_name: str,
+        node_pool_name: str,
+        *,
+        token: str | None = None,
+    ) -> NodePool:
+        assert self._client
+        url = (
+            self._clusters_url
+            / cluster_name
+            / "cloud_prodider"
+            / "node_pools"
+            / node_pool_name
+        )
+
+        headers = self._create_headers(token=token)
+        async with self._client.get(url=url, headers=headers) as responce:
```

Comment:
typo, response

Reply:
fixed in [4b49d1a](https://github.com/neuro-inc/neuro-config-client/pull/83/commits/4b49d1afae1854dfa25aa96e86fa5cbd1bed6224)